tests/quantization/test_compressed_tensors.py

```diff
@@ -11,8 +11,10 @@
 
 
 def test_compressed_tensors_w8a8_static_setup(vllm_runner):
-    model_path = "nm-testing/tinyllama-one-shot-static-quant-test-compressed"
-    llm = vllm_runner(model_path, quantization="sparseml", enforce_eager=True)
+    model_path = "nm-testing/tinyllama-oneshot-w8a8-static-v2"
+    llm = vllm_runner(model_path,
+                      quantization="compressed-tensors",
+                      enforce_eager=True)
```

Comment:
Is it necessary to force the quantization config here - it should auto-detect right? Also do we have a test without eager forced?

Reply:
+1 I would like to make sure this auto-detects

Reply:
Updated to auto detect